src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskLoader.scala

```diff
@@ -0,0 +1,9 @@
+package mesosphere.marathon.core.task.tracker.impl
+import scala.concurrent.Future
+
+/**
+  * Loads all task data into an [[AppDataMap]].
+  */
+private[tracker] trait TaskLoader {
```

Comment:
What would you suggest instead? Just mock the Impl directly? I like this approach better -- not much more work -- but I do not care that much. If you want to, I can change this.